lisp/pdf-view.el

```diff
@@ -1219,6 +1220,45 @@ The colors are determined by the variable
   (pdf-cache-clear-images)
   (pdf-view-redisplay t))
 
+(defun pdf-view-refresh-all-themed-buffers (&optional args)
```

Comment:
args is never used.

Reply:
Note that this arg just captures the argument passed to `enable-theme` from the :after advice for `enable-theme` and `disable-theme`. Since this function is also called from `pdf-view-set-theme-background` I think it would be more complicated for it to be :around advice and thus take a function argument and argument list rather than an unused argument. I added extra documentation to clarify this in 73508f7.